README.md

```diff
@@ -92,26 +92,74 @@ Two specific points to expand on:
 
 ### Setting up
 
-1. Follow the [Flutter installation guide](https://docs.flutter.dev/get-started/install)
-   for your platform of choice.
-2. Switch to the latest version of Flutter by running `flutter channel main`
-   and `flutter upgrade` (see [Flutter version](#flutter-version) below).
-3. Ensure Flutter is correctly configured by running `flutter doctor`.
-4. Start the app with `flutter run`, or from your IDE.
+1. Setup the pinned version of flutter.
+   (See [Flutter version](#flutter-version) for details on what this
+    step does and read it if you want to use your own version of
+    flutter.)
+
+    1.1. Make sure you have initialized submodule by doing (do
+  not do `--depth=1` as flutter relies on tags to know its version):
```

Comment:
[[[WP:BEANS]]](https://en.wikipedia.org/wiki/WP:BEANS) — simpler just not to mention `--depth` :slightly_smiling_face: